### ISSUE TITLE
css property 'color' should not be defined in .vue-info

### ISSUE THREAD

**Issue by KdavisO** [pocka/storybook-addon-vue-info#55]
hello sir
may I ask you to slove below probrem?

**Describe the bug**

```
The `color` you define in `.vue-info` will override my storybook's style. I think it is better to remove.
(all color of original components become `#333`,,,)
https://github.com/pocka/storybook-addon-vue-info/blob/master/src/components/InfoView.vue#L103
```

I'm waiting for your good answer.
thank you

**Comment by pocka** [comment]
Thank you for reporting issue!

> I think it is better to remove.

Agreed. We should style each child component, not wrapper component (Sorry, this is because my laziness 😞 ). 
I'll fix this in v0.6.2.

**Comment by KdavisO** [comment]
Thanks for your very immediate response!
I'm looking forward to v0.6.2. release.

Thank you indeed!!
